lib/mayaUsd/ufe/Global.cpp

```diff
@@ -358,6 +367,9 @@ MStatus initialize()
 #if UFE_LIGHTS_SUPPORT
     runTimeMgr.setLightHandler(usdRtid, UsdLightHandler::create());
 #endif
+#if UFE_LIGHTS2_SUPPORT
+    runTimeMgr.setLight2Handler(usdRtid, UsdLight2Handler::create());
+#endif
```

Comment:
I don't think you need this. Your new light2 is only in Ufe v6 and above, so that implies ufe v4 so it won't enter the ifndef on line 365.